algorithms/beefy/prover/src/lib.rs

```diff
@@ -307,7 +307,7 @@ where
 			signed_commitment.commitment.block_number.into();
 		let block_hash = self.relay_client.rpc().block_hash(Some(subxt_block_number)).await?;
 
-		let current_authorities = api.storage().beefy().authorities(block_hash).await?.0;
+		let current_authorities = api.storage().beefy().authorities(block_hash).await?;
```

Comment:
This isn't compatible with polkadot v0.9.27

Reply:
It's strange, because this code wasn't compiling